charts/rancher-cis-benchmark/0.1.2/Chart.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+appVersion: "0.1.2"
+description: |
+  Run CIS benhmark tests
```

Comment:
There is a typo here, but we can merge ahead. @prachidamle can you fix this in the dev-v2.4 branch for the next release?